### PR TITLE
fix phpDoc and broken return type in Common/Select.php

### PR DESCRIPTION
**Pull request by pavarnos**

### CODE REVIEW

File: src/Common/Select.php [hunk 1]
```diff
@@ -248,8 +248,6 @@ protected function addCol($key, $val)
      * @param string $spec The column specification: "col alias",
      * "col AS alias", or something else entirely.
      *
-     * @return null
```
Comment (harikt) [inline src/Common/Select.php:251]: So what is the return type we can define ? void ?

File: src/Common/Select.php [hunk 2]
```diff
@@ -273,7 +271,7 @@ protected function addColWithAlias($spec)
      *
      * @param string $alias The column to remove
      *
-     * @return null
+     * @return boolean
```
Comment (harikt) [inline src/Common/Select.php]: Inconsistency in return type : https://github.com/auraphp/Aura.SqlQuery/pull/110/files#diff-57cf575fdd7a21c19685563c9fa2a1c0L300

File: src/Common/Select.php (hunk 1 above)
Comment (pavarnos) [reply src/Common/Select.php:251]: I was fixing a phpStorm lint warning that said "missing return statement". Didn't notice all the other similar occurrences in the file and throughout the code. I guess for consistency, it should `@return null` or `@return void` everywhere. What is your preference?

Comment (harikt) [reply src/Common/Select.php:251]: return null is all good for now I believe.

Thank you

File: src/Common/Select.php (hunk 2 above)
Comment (harikt) [reply src/Common/Select.php]: This is not yet fixed. Why not use `bool` ?

Comment (pavarnos) [reply src/Common/Select.php]: bool and boolean are the same, (http://php.net/manual/en/language.types.boolean.php). Changed it to bool. The old return type of null was incorrect: every return value is either `true` or `false`.

**Comment by harikt** [comment]
Thank you.

**Pull request merged**